backend/test/unit/controller/levelController.test.ts

```diff
@@ -0,0 +1,92 @@
+import { expect, test, jest, afterEach } from '@jest/globals';
+import { Response } from 'express';
+
+import { handleLoadLevel } from '@src/controller/levelController';
+import { LevelGetRequest } from '@src/models/api/LevelGetRequest';
+import { ChatModel } from '@src/models/chat';
+import { ChatMessage } from '@src/models/chatMessage';
+import { Defence } from '@src/models/defence';
+import { EmailInfo } from '@src/models/email';
+import { LEVEL_NAMES } from '@src/models/level';
+
+declare module 'express-session' {
+	interface Session {
+		initialised: boolean;
+		chatModel: ChatModel;
+		levelState: LevelState[];
+	}
+	interface LevelState {
+		level: LEVEL_NAMES;
+		chatHistory: ChatMessage[];
+		defences: Defence[];
+		sentEmails: EmailInfo[];
+	}
+}
+
+jest.mock('@src/promptTemplates', () => ({
+	systemRoleLevel1: 'systemRoleLevel1',
+	systemRoleLevel2: 'systemRoleLevel2',
+	systemRoleLevel3: 'systemRoleLevel3',
+}));
+
+const mockSend = jest.fn();
+
+function responseMock() {
+	return {
+		send: mockSend,
+		status: jest.fn().mockReturnValue({ send: mockSend }),
+	} as unknown as Response;
+}
+
+afterEach(() => {
+	mockSend.mockClear();
+});
+
+test('WHEN client asks to load level 1 THEN the backend sends the level information for the given level', () => {
```

Comment:
So the level number itself is unimportant? Or is it that the levelState array only has one entry in it, hence we return the first entry for level 1? If the latter is true, then it'd be good to have tests ensuring all our levels are covered so we know we are returning the correct level state, or at least that level1 and sandbox are returned, and also that a level outside of bounds returns a 400.